### PR TITLE
Docusaurus heading component

### PR DESCRIPTION
**Pull request by Yolijn**
- [x] Add components to docusaurus
- [x] Improve heading and sidebars for docusaurus
- [x] Separate bem/css stories to enable neat iframes for each story

### CODE REVIEW

File: standalone/docusaurus/src/components/HomepageFeatures.js [hunk 1]
```diff
@@ -1,11 +1,14 @@
+import Bicycles from '@site/static/img/bicycles.png';
+import Laptop from '@site/static/img/laptop.png';
+import Utrecht from '@site/static/img/utrecht.png';
```
Comment (Yolijn) [inline standalone/docusaurus/src/components/HomepageFeatures.js:3]: By importing images like this, it doesn't break when the BaseUrl is changed at any point

File: .eslintignore [hunk 2]
```diff
@@ -1,9 +1,11 @@
 # Ignore 3rd party files
 node_modules
+iframeResizer.contentWindow.min.js
```
Comment (Robbert) [inline .eslintignore:3]: We could consider including `*.min.js` in "generated files"

File: standalone/docusaurus/docs/componenten/Heading.mdx [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+  - h2
+  - h3
+  - h4
+  - h5
+  - h6
+  - heading
+  - utrecht-heading
+---
+
+import { Markdown } from "../../src/components/Markdown";
```
Comment (Robbert) [inline standalone/docusaurus/docs/componenten/Heading.mdx:20]: I have tried to include `<!-- @license CC0-1.0 -->` in every `*.md`/`*.mdx` documentation page, so it easy to copy to other repos while maintaining the open source license and it is clear that the license for docs is different from the code repository overall.

File: components/heading-4/bem.stories.mdx [hunk 4]
```diff
@@ -0,0 +1,60 @@
+<!-- @license CC0-1.0 -->
+
+import { ArgsTable, Canvas, Meta, Story } from "@storybook/addon-docs";
+import { ComponentTokensSection } from "../../documentation/components/ComponentTokensSection";
+import tokens from "../../proprietary/design-tokens/dist/index.json";
+import "../heading-4/bem.scss";
```
Comment (Robbert) [inline components/heading-4/bem.stories.mdx:6]: I kind of expected the import order ESLint plugin to complain that the `.css` import (side effect imports) should be the last.

**Review by Robbert — APPROVED** [review]

File: .eslintignore (hunk 2 above)
Comment (Yolijn) [reply .eslintignore]: As discussed, let's add another section for "Ignore minified files" as it's not generated by us and it's probably better not to make future developers assume something different :)

File: components/heading-4/bem.stories.mdx (hunk 4 above)
Comment (Yolijn) [reply components/heading-4/bem.stories.mdx]: The import order in mdx files isn't checked by our linting. But I've moved those side-effect imports to be the last import of the heading mdx files

**Pull request merged**